api/openapi-spec/swagger.json

```diff
@@ -18400,6 +18400,16 @@
      }
     }
    },
+   "v1alpha1.VirtualMachinePoolDownscalePolicy": {
```

Comment:
Can you use the abstraction from this [design document](https://github.com/kubevirt/community/blob/main/design-proposals/vm-pool.md#virtualmachinepool-vmpool-api)?  I think your feature fits into the API like this: 
`.ScaleInStrategy.Proactive.SelectionPolicy.BasePolicy = ordered`
`.ScaleInStrategy.Proactive.SelectionPolicy.BasePolicy = random`